custom_components/tahoma/entity.py

```diff
@@ -88,40 +89,28 @@ def device_info(self) -> dict[str, Any]:
             or self.device.widget
         )
 
-        return {
-            "identifiers": {(DOMAIN, self.executor.base_device_url)},
-            "manufacturer": manufacturer,
-            "name": self.device.label,
-            "model": model,
-            "sw_version": self.device.controllable_name,
```

Comment:
I would like to see the device.controllable_name somewhere. It helps me do retrieve the setup.

Reply:
Same! I would propose to add this to system_health and update our README? Or only keep this in the debug logging and ask people to add their debug logs.

It doesn't make a lot of sense to have this visible in the main interface, if we only need it for debug purposes. Agree?

Reply:
Agree! Can you add the debug log with this PR?

Reply:
@tetienne don't we already have all this information in the debug log? Only system health isn't in this PR yet.